modules/test/playwright/tests/portal-web/portal.spec.ts

```diff
@@ -3,15 +3,15 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import {expect, test} from '@playwright/test';
+import {Page, expect, test} from '@playwright/test';
 
-test('title is Home - Liferay DXP', async ({page}) => {
+test('title is Home - Liferay DXP', async ({page}: {page: Page}) => {
```

Comment:
Are you sure this is really needed? Vim+CoC autoinfers the `Page` type in my env... :thinking:

Reply:
Hmm yeah, you're right, it works by some magic in vscode as well. If it's not necessary, no need to put it in. I'll revert the changes.